hyperlink/test/test_scheme_registration.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+
+from .common import HyperlinkTestCase
+from .._url import register_scheme, URL
+
+
+class TestSchemeRegistration(HyperlinkTestCase):
+
+    def test_register_scheme_basic(self):
+        register_scheme('deltron', uses_netloc=True, default_port=3030)
+
+        u1 = URL.from_text('deltron://example.com')
+        assert u1.scheme == 'deltron'
```

Comment:
Per an earlier discussion, the TestCase element is an artifact of leaving the old twisted.python.url test cases as stable as can be. I'm planning a pytest migration very soon, once other aspects settle down. These tests will port right over.